frontend/src/features/SideWindow/MissionForm/hooks/useListenMissionEventUpdates.ts

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useRef, useState } from 'react'
+
+import { MONITORENV_API_URL } from '../../../../api/api'
+import { Mission } from '../../../../domain/entities/mission/types'
+import { useMainAppSelector } from '../../../../hooks/useMainAppSelector'
+import { missionEventListener } from '../sse'
+
+const MISSION_UPDATES_URL = `${MONITORENV_API_URL}/api/v1/missions/sse`
+const MISSION_UPDATE_EVENT = `MISSION_UPDATE`
+
+export function useListenMissionEventUpdates() {
```

Comment:
`useListenToEditedMissionEventUpdates()` ?

Reply:
+1 pour le "[...]To[...]", par contre je ne sais pas si "Edited" apporte grand-chose, surtout s'il est retiré dans "[...]ById"